web/shapeworks/src/components/NavBar.vue

```diff
@@ -19,56 +20,42 @@ export default defineComponent({
             }
         }
 
+        function toSelectPage(){
+          router.push({
+              name: 'select',
+          });
+        }
+
         return {
             oauthClient,
             params,
             logInOrOut,
+            toSelectPage,
             selectedDataset,
         }
     }
 })
 </script>
 
 <template>
-  <v-app-bar app height="80px">
+  <v-app-bar app height="50px">
     <div class="d-flex align-center px-5">
       <v-img
         alt="ShapeWorks Logo"
         src="favicon.ico"
         transition="scale-transition"
         width="55px"
       />
-      <v-toolbar-title class="text-h6">ShapeWorks</v-toolbar-title>
+      <v-toolbar-title class="text-h6" @click="toSelectPage">ShapeWorks</v-toolbar-title>
```

Comment:
```suggestion
      <v-toolbar-title class="text-h6" to="select">ShapeWorks</v-toolbar-title>
```
Vue Router patches the `to` prop onto some elements, but possibly not `v-toolbar-title`. If that doesn't work:
```suggestion
      <v-toolbar-title class="text-h6"><router-link to="select">ShapeWorks</router-link></v-toolbar-title>
```

Reply:
Changed in [8c9dfb8](https://github.com/girder/shapeworks-cloud/pull/146/commits/8c9dfb881a8fe124f9421878d8875ef60376642c). You were right that the `to` attribute didn't work, but I was able to use a router link and override the css that made it blue and underlined. I decided to put the router link around the whole logo-text flexbox so that either the logo or the text may be clicked to navigate.